.github/workflows/docker-ci.yml

```
@@ -0,0 +1,52 @@
+name: Docker CI
+
+on:
+  push:
+    branches:
+      - 'chore-add-image-build-test-to-github-actions'
+
+jobs:
+  build_and_test:
+    name: Build and Test Docker Image on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
```

Please specify a reasonable timeout:

```suggestion
    runs-on: ${{ matrix.os }}
    timeout-minutes: 15
```